src/resources/views/post/index.blade.php

```diff
@@ -21,10 +21,13 @@
                 @if ($post->image)
                     <img height="250px" width="150px" src="{{ asset('/storage/images/'.$post->image) }}" alt="Post Image" class="img-fluid">
                 @endif
+                <span class="like-count" data-post-id="{{ $post->id }}">
+                    {{ $post->likes->count() === 0 ? '' : $post->likes->count() . ' ' . ($post->likes->count() === 1 ? 'Like' : 'Likes') }}
```

Comment:
For future references: You can move this to your controller. Try to minimize logic handling in your blade files.